packages/terra-form-input/src/terra-dev-site/doc/example/controlled/InputProgrammaticSet.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useRef } from 'react';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import Input from 'terra-form-input';
+import Styles from './examplestyles.scss';
+
+const cx = classNames.bind(Styles);
+
+const InputProgrammaticSet = () => {
+  const [text, setText] = useState('');
+  const inputRef = useRef(null);
+
+  const setTextClick = () => {
+    if (inputRef.current) {
+      inputRef.current.value = 'IpsumLorem';
+      let customEvent;
+      if (typeof (Event) === 'function') {
+        customEvent = new Event('input', {
+          bubbles: true,
+          cancelable: false,
+        });
+      } else {
+        customEvent = document.createEvent('Event');
+        customEvent.initEvent('input', true, false);
+      }
+      inputRef.current.dispatchEvent(customEvent);
+    }
+  };
+
+  const handleChange = (event) => {
+    setText(event.target.value);
+  };
+
+  const handleInput = (event) => {
+    setText(event.target.value);
+  };
+
+  return (
+    <div>
+      <Input
+        onChange={handleChange}
+        onInput={handleInput}
+        value={text}
+        refCallback={(ref) => { inputRef.current = ref; }}
+      />
+      <Button text="Set input text" onClick={setTextClick} id="input-set-text-button" className={cx('programattic-button')} />
```

Comment:
Small typo in the classname, `programattic` -> `programmatic`

Reply:
Updated here https://github.com/cerner/terra-core/pull/3135/commits/17a5bc400687a7f9b1b945eb45b51f42adbb27f6